scripts/helmlint.sh

```diff
@@ -12,4 +12,18 @@ TARGET_DIR="${TEMP_DIR}/kubecf"
 mkdir "${TARGET_DIR}"
 
 tar xfz "${TARGET_FILE}" -C "${TEMP_DIR}"
-helm lint "${TARGET_DIR}"
+# Workaround: [Helm lint (v3) doesn't take values files into account during schema validation]
+# (https://github.com/helm/helm/issues/7756). Fixed in helm 3.3.0.
```

Comment:
Yes, I'll change the minimum version to 3.3.0. It was just released within the last 24 hours...

Reply:
Requiring helm 3.3.0 in 87f52bd. Needed some additional changes because there is still some weirdness about overriding a nil value, and lint becoming more picky about empty YAML documents.